client/src/components/UnsavedChangesIndicator/UnsavedChangesIndicator.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import i18n from 'i18n';
+import { withUnsavedChangesIndicatorTimer } from './UnsavedChangesIndicatorTimer';
+
+const showComponent = (level) => ['notice', 'warning'].includes(level);
+
+const UnsavedChangesIndicator = ({
+  level = 'none',
+  elapsedMinutes = 0,
+}) => {
+  const [ariaLive, setAriaLive] = useState('off');
+  // Use a ref instead of state to track previous level without causing re-renders
+  const prevLevelRef = useRef(level);
+
+  useEffect(() => {
+    if (!showComponent(level)) {
+      prevLevelRef.current = level;
+      return;
+    }
+    if (level !== prevLevelRef.current) {
+      setAriaLive('polite');
+      prevLevelRef.current = level;
+    } else {
+      setAriaLive('off');
+    }
+  }, [level, elapsedMinutes]);
+
+  if (!showComponent(level)) {
+    return null;
+  }
+  let message;
+  let shortMessage;
+  // String '0' is required for i18n.inject() to work correctly if using a value of 0
+  const mins = elapsedMinutes || '0';
+  if (mins <= 1) {
+    message = i18n.inject(
+      i18n._t('Admin.UNSAVED_CHANGES_SINGULAR_FULL', 'Unsaved changes: {mins} min'),
+      { mins }
+    );
+    shortMessage = i18n.inject(
+      i18n._t('Admin.UNSAVED_CHANGES_SINGULAR_SHORT', '{mins} min'),
+      { mins }
+    );
+  } else {
```

Comment:
As per https://github.com/silverstripe/silverstripe-admin/pull/2058#discussion_r2540885374 we're not expecting such low values - I think given this is an unlikely configuration, it's fine to just ignore this scenario. "One minutes" isn't the worst thing in the world to see in the unlikely scenario it does happen.

Even if we did keep this, at least in english "zero minutes" is correct, and "zero minute" is not, and since our JS i18n has no pluralisation support trying to handle this for other languages is a non-starter.

Reply:
Have changed `<= 1` to `=== 1`